index.html

```diff
@@ -50,6 +51,7 @@
   <body>
```

Comment:
Apply tailwind custom background color class to body. "primarybg"
```suggestion
  <body class="primarybg">
```